### DISCUSSION TITLE
Is expo-updates can update native code?

### DISCUSSION THREAD

**Discussion by SnowingFox** [expo/expo#26139] [Archived: Questions] [answered]
In Expo development mode, we can configure our native code in `android` or `ios` folder

let's assume that we have two version 1.0 & 1.1 in IOS

- In 1.0, it's a clear project that don't have extra native package
- but in 1.1, we import a native package like "expo-av“, and we will update Pods and install pod package

My question is in 1.1 version we can use expo-updates update application with native packages？

**Comment by Jeroen-G** [comment, marked as answer]
This page has a nice visual that explains that in this case you can not do an OTA update: https://docs.expo.dev/eas-update/how-it-works/
